src/FINDOLOGIC/Export/Data/Usergroup.php

```diff
@@ -13,7 +15,7 @@ class Usergroup implements Serializable
 
     public function __construct($value)
     {
-        $this->value = DataHelper::checkForEmptyValue($value);
+        $this->value = DataHelper::checkForEmptyValue('usergroup', $value);
```

Comment:
Can we implement the `NameAwareValue` class here as well and access the name via the `getValueName()` method instead?